src/org/traccar/model/Event.java

```diff
@@ -56,6 +56,8 @@ public Event() {
     public static final String TYPE_IGNITION_ON = "ignitionOn";
     public static final String TYPE_IGNITION_OFF = "ignitionOff";
 
+    public static final String TYPE_MAINTENANCE_NEED = "maintenanceNeed";
```

Comment:
I think we should just name it "maintenance" or alternatively "maintenance needed" (note -ed).